.github/workflows/test.yml

```diff
@@ -240,9 +240,36 @@ jobs:
       # UNIT TESTS FRONTEND ####################################################
       ##########################################################################
       - name: frontend | Unit tests
-        run: docker run --rm gradido/frontend:test yarn run test
-
-
+        run: docker run -v /coverage:/app/coverage --rm gradido/frontend:test yarn run test
+      ##########################################################################
+      # COVERAGE CHECK FRONTEND ################################################
+      ##########################################################################
+      #- name: frontend | Coverage check
+      #  uses: devmasx/coverage-check-action@v1.2.0
+      #  with:
+      #    type: simplecov
+      #    result_path: /coverage/coverage-final.json
+      #    min_coverage: 10
+      #    token: ${{ github.token }}
+      ##########################################################################
+      # COVERAGE REPORT FRONTEND ################################################
+      ##########################################################################
+      #- name: show content of /coverage/lcov.info
+      #  run: cat /coverage/lcov.info
+      #- name: frontend | coverage report
+      #  uses: romeovs/lcov-reporter-action@v0.2.16
+      #  with:
+      #    github-token: ${{ secrets.GITHUB_TOKEN }}
+      #    lcov-file: /coverage/lcov.info
+      ##########################################################################
+      # COVERALLS REPORT FRONTEND ##############################################
+      ##########################################################################
+      - name: frontend | Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          path-to-lcov: /coverage/lcov.info
```

Comment:
Can we please split this in its own job? So its distinguishable from the unit tests?

Reply:
![image](https://user-images.githubusercontent.com/1238238/114475656-f506f280-9bf8-11eb-9b4c-a63bfe6e0062.png)

Also why PR & push?

Where is the coverage report?